src/plot_api/index.js

```diff
@@ -16,6 +16,8 @@ exports.restyle = main.restyle;
 exports.relayout = main.relayout;
 exports.redraw = main.redraw;
 exports.update = main.update;
+exports.decode = main.decode;
+exports.encode = main.encode;
```

Comment:
We may start with underscore (i.e. kind of private) methods here.
For example:
```js
exports._decode = main.decode;
exports._encode = main.encode;
```
If we want to expose these functionality, then using a more specific name could be considered.
For example: 
```js
exports.decodeArray = main.decode;
exports.encodeArray = main.encode;
```